pkg/scheduler/framework/plugins/nodeaffinity/node_affinity_test.go

```diff
@@ -1234,8 +1234,11 @@ func TestNodeAffinityPriority(t *testing.T) {
 }
 
 func Test_isSchedulableAfterNodeChange(t *testing.T) {
+	qhintEnabled := true
+	qhintDisabled := false
 	podWithNodeAffinity := st.MakePod().NodeAffinityIn("foo", []string{"bar"}, st.NodeSelectorTypeMatchExpressions)
 	testcases := map[string]struct {
+		qhints         *bool
```

Comment:
You could try to make it similar to other tests (see [here](https://github.com/kubernetes/kubernetes/blob/8adc35ebd7221683b9f9ec7a9e2c63cf3f6d0dff/test/integration/scheduler/queue_test.go#L204-L206)), so we won't need `qhintEnabled` and `qhintDisabled` variables then.